src/modules/sensors/data_validator/DataValidatorGroup.hpp

```diff
@@ -141,7 +141,7 @@ class DataValidatorGroup
 	int _prev_best{-1}; /**< the previous best index */
 
 	uint64_t _first_failover_time{0}; /**< timestamp where the first failover occured or zero if none occured */
-
+	int8_t _last_failover_index{-1}; /**< index of the last failover */
```

Comment:
```suggestion
	int8_t _last_failover_index{-1};
```
what else could it be?

Reply:
Please don't follow that bat habit of commenting every line https://youtube.com/clip/UgkxcPLMW6uiLRvBmoKkDP7jm0ntYxDyaxLS?si=TKwknK0o-JEvR7cN